cms_status_message/models.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# © 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models, api
+from openerp.http import request
+from openerp import _
```

Comment:
usually I don't do more than one import on one line, because it's easier to copy/paste what you need and above all to spot import issues. But I can merge them if needed.